diff -Nuar tbc-9999.orig/pym/tbc/sqlquerys.py tbc-9999/pym/tbc/sqlquerys.py
--- tbc-9999.orig/pym/tbc/sqlquerys.py	2016-12-26 23:10:55.724000000 +0000
+++ tbc-9999/pym/tbc/sqlquerys.py	2016-12-26 23:11:24.529000000 +0000
@@ -636,7 +636,7 @@
 		for x in EMInfo:
 			x.New = False
 			session.flush()
-try:
+	try:
 		BLInfo = session.query(BuildLogs).filter(BuildLogs.New == True).all()
 	except NoResultFound as e:
 		pass
